build.gradle.kts

```diff
@@ -15,8 +16,8 @@ repositories {
 dependencies {
     implementation("com.android.tools.build:gradle:4.0.0")
 
-    testImplementation("io.kotlintest:kotlintest-runner-junit5:3.1.10")
-    testImplementation("junit:junit:4.13")
+    testImplementation("io.kotlintest:kotlintest-runner-junit5:3.1.11")
```

Comment:
I think kotlintest is renamed to kotest. Maybe this should be changed to kotest?

Reply:
Thanks but the rename is only for release 4.x, the project is not yet compatible for this version so I will let kotlintest in 3.1.11 until a real migration.
https://github.com/kotest/kotest/blob/master/doc/reference.md